packages/app/src/runner/event-manager.ts

```diff
@@ -597,29 +593,33 @@ export class EventManager {
     })
 
     Cypress.on('test:before:run:async', async (...args) => {
-      const [attr, test] = args
+      const [attributes, test] = args
 
-      this.reporterBus.emit('test:before:run:async', attr)
+      this.reporterBus.emit('test:before:run:async', attributes)
 
       this.studioStore.interceptTest(test)
 
       // if the experimental flag is on and we are in a chromium based browser,
       // check the memory pressure to determine if garbage collection is needed
       if (Cypress.config('experimentalMemoryManagement') && Cypress.isBrowser({ family: 'chromium' })) {
         await Cypress.backend('check:memory:pressure', {
-          test: { title: test.title, order: test.order, currentRetry: test.currentRetry() },
+          test: { title: attributes.title, order: attributes.order, currentRetry: attributes.currentRetry },
         })
       }
 
-      await Cypress.backend('protocol:test:before:run:async', { id: test.id, title: test.title, wallClockStartedAt: test.wallClockStartedAt.getTime() })
+      await Cypress.backend('protocol:test:before:run:async', attributes)
```

Comment:
I think I had tried this earlier, and attributes can be enormous and/or recursive. We may want to be more targeted about this.

Reply:
Passing the attributes to the protocol code should be fine as long as we don't try to save all the attributes.

Reply:
Spoke with @ryanthemanuel and he had previously passed the entire `test` runnable and not the `attributes`.